snafu/uperf_wrapper/uperf_wrapper.py

```diff
@@ -54,6 +54,13 @@ def __init__(self, parent_parser):
         self.args.num_pairs = os.getenv("num_pairs", "")
         self.args.multus_client = os.getenv("multus_client", "")
         self.args.networkpolicy = os.getenv("networkpolicy", "")
+        self.args.nodes_in_iter = os.getenv("node_count", "0")
```

Comment:
I think we should just set it to empty as default, any reason or 0s? Does ES complain if its an empty field?

Reply:
I was going for a sentinel value, that we would know if it wasn't set in the config file. Empty string works as well, but I figured ES returning results with a value that is known as an error code would be useful.  Will change.